godtools/Views/TractElements/TractElementFunctions.swift

```diff
@@ -93,16 +93,20 @@ extension BaseTractElement {
         return nil
     }
     
-    func sendMessageToElement(tag: String) {
-        if TractBindings.bindings[tag] != nil {
-            guard let view = TractBindings.bindings[tag] else { return }
+    func sendMessageToElement(listener: String) {
+        if TractBindings.bindings[listener] != nil {
```

Comment:
The reason why I did this is because there are other conditions, so if it returns it will not check the other conditions, and, just in case, there may be a case where the listener appear on different places, on `listeners` and `dismiss-listeners` attributes